### PR TITLE
Disable mute button if media is not yet available. 

### PR DESCRIPTION
**Pull request by toger5**
This also fixes the jumpy mute unmute animation which can be observed at the very start of the lobby view.
Screenshot 2023-07-10 at 11 41 48

### CODE REVIEW

File: src/button/Button.module.css [hunk 1]
```diff
@@ -61,6 +61,17 @@ limitations under the License.
...
 }
 
+.button:disabled,
+.toolbarButton:disabled,
+.toolbarButtonSecondary:disabled,
+.iconButton:disabled,
+.iconCopyButton:disabled,
+.secondary:disabled,
+.secondaryHangup:disabled,
+.copyButton:disabled {
```
Comment (SimonBrandner) [inline src/button/Button.module.css]: Do we need to list these all? Shouldn't they all have class of `.button`?

**Review by SimonBrandner — APPROVED** [review]
Looks good otherwise

File: src/button/Button.module.css (hunk 1 above)
Comment (toger5) [reply src/button/Button.module.css]: good question. this is copied from the focus field above.

Comment (toger5) [reply src/button/Button.module.css]: also works with just button.
The question is, if we really want all buttons to behave like this...

Comment (toger5) [reply src/button/Button.module.css]: I think for now to just have this style for the toolbarButton might be the best option to avoid style regressions.

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**